AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -168,6 +168,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57596
 br0wsers.com,softwaresde.com#%#//scriptlet("adjust-setInterval", "show_ag", "", "0.02")
+! https://github.com/AdguardTeam/AdguardFilters/issues/58321
+an1.com#%#AG_onLoad(function(){ countdown(0); window.countdown = {}; });
```

Comment:
Good, but what about scriptlets?
Something like that:
`an1.com#%#//scriptlet('adjust-setTimeout', 'countdown', '1400')`

Reply:
The scriptlet works fine on my end.